staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -241,7 +241,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	APIServerTracing: {Default: true, PreRelease: featuregate.Beta},
 
-	ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Alpha},
+	ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Beta},
```

Comment:
same comment about whether this is going to fix the remaining items on the checklist.

Reply:
reverted